cli/boards.go

```diff
@@ -311,56 +305,6 @@ func sendPackageRequests(stream packagepb.PackageService_CreatePackageClient,
 	return nil
 }
 
-// createArchive creates a tar.gz from the file provided.
-func createArchive(file *os.File) (*bytes.Buffer, error) {
-	// Create output buffer
-	out := new(bytes.Buffer)
-
-	// These writers are chained. Writing to the tar writer will
-	// write to the gzip writer which in turn will write to
-	// the "out" writer
-	gw := gzip.NewWriter(out)
-	defer utils.UncheckedErrorFunc(gw.Close)
-	tw := tar.NewWriter(gw)
-	defer utils.UncheckedErrorFunc(tw.Close)
-
-	// Get FileInfo about our file providing file size, mode, etc.
-	info, err := file.Stat()
-	if err != nil {
-		return nil, err
-	}
-
-	// the raw file can be 100 times more than the max TAR size.
-	if info.Size() > 100*boardUploadMaximumSize {
```

Comment:
Is this limit no longer relevant (I think you mentioned this in another comment but wanted to verify)?